### ISSUE TITLE
[Issue]: build with dotnet sdk 6.0.400 fails with NETSDK1067

### ISSUE THREAD

**Issue by hgy59** [jellyfin/jellyfin#8245]
### Please describe your bug

the build of jellyfin 10.8.3 fails with 
```
sdk/6.0.400/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.RuntimeIdentifierInference.targets(157,5): error NETSDK1067: Self-contained applications are required to use the application host. Either set SelfContained to false or set UseAppHost to true.
```

Probably this is an issue of the latest dotnet sdk, but it might be related to the jellyfin build.

### Jellyfin Version

Other

### if other:

10.8.3

### Environment

```markdown
- OS:
- Virtualization:
- Clients:
- Browser:
- FFmpeg Version:
- Playback Method:
- Hardware Acceleration:
- Plugins:
- Reverse Proxy:
- Base URL:
- Networking:
- Storage:
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by hgy59** [comment]
BTW: jellyfin builds successfullly with dotnet SDK 6.0.301 and 6.0.302.

**Comment by hgy59** [comment]
I figured out that the publish parameter `"-p:DebugSymbols=false;DebugType=none;UseAppHost=true"` does not work anymore.

We used different configuration parameters that lead to the error reported above `"-p:UseAppHost=true;PublishReadyToRun=true;PublishReadyToRunShowWarnings=true"`

anyway using unquoted parameters, each prefixed with `-p:` solves this issue.

either kind is working without errors:
- `-p:DebugSymbols=false -p:DebugType=none -p:UseAppHost=true`
- `-p:UseAppHost=true -p:PublishReadyToRunShowWarnings=true -p:PublishReadyToRun=true`

---

I suppose that the parameter separator `;` does not work since spk 6.0.400, as the error reported with `"-p:DebugSymbols=false;DebugType=none;UseAppHost=true"` is 
`error MSB4030: "false;DebugType=none;UseAppHost=true" is an invalid value for the "EmitDebugInformation" parameter of the "Csc" task. The "EmitDebugInformation" parameter is of type "System.Boolean".`
